Library/DataSource/UICollectionView-Extensions.swift

```diff
@@ -1,24 +1,27 @@
 import UIKit
 
 public extension UICollectionView {
+  // MARK: - Registration
+
   func registerCellClass<CellClass: UICollectionViewCell>(_ cellClass: CellClass.Type) {
-    self.register(cellClass, forCellWithReuseIdentifier: cellClass.description())
+    self.register(cellClass, forCellWithReuseIdentifier: classNameWithoutModule(cellClass))
```

Comment:
Needed to use some of these convenience functions so in regards to the plan to merge feature branch soon I haven't made these changes against `master` first (only here).